server/routes/admin/html.py

```diff
@@ -47,8 +47,12 @@ def load_data():
 
   # TODO: dynamically create the output dir.
   output_dir = os.path.join(user_data_path, 'datacommons')
+  # TODO: (now very brittle) Should make this NL path globally sharable by
+  # - nl server
+  # - web server (admin portal)
+  # - build embeddings tool
   nl_dir = os.path.join(output_dir, 'nl')
-  sentences_path = os.path.join(nl_dir, 'sentences.csv')
+  nl_embeddings_dir = os.path.join(nl_dir, 'embeddings')
```

Comment:
Any reason to use a separate folder? Changing the folder here will require changes to the config reader to read from this new folder and will also need changes to @ajaits's scripts in https://github.com/datacommonsorg/website/pull/4268

Reply:
config reader is fine since that depends on the custom_catalog.yaml `embeddings_path` which is set directly. 

I have also commented on some pros of using directory than a single file name. We prob should sync with Ajai on using directory as well.

Reply:
Would the path to `custom_catalog.yaml` itself need to be changed here? https://github.com/datacommonsorg/website/blob/7accd94de4edc21e3872603839f3ffaa001fb457/nl_server/config_reader.py#L55

Reply:
this is okay. note in current change, building_custom_embeddings.yaml, there is

`local_catalog_path = os.path.join(fm.local_input_dir(), _CATALOG_FILE_NAME)`

so the custom_catalog.yaml is still under the input dir. 

But this will be very quickly changed in the next PR anyway..

Reply:
OK. Not fully sure what the end folder structure looks like but as long as it works and cleaned up in the follow up PR, we're good.